### ISSUE TITLE
Missing Type.GetTypeCode

### ISSUE THREAD

**Issue by Muchiachio** [dotnet/coreclr#1863]
Will the `Type.GetTypeCode()` be supported in core clr?
I can't find it neither in `Type` nor in `TypeInfo` declarations.

**Comment by weshaggard** [comment]
cc @pallavit @nguerrera

**Comment by neyromant** [comment]
I have same question.
For example, I want write this code in coreclr (Check type is primitive): 
```csharp
private static bool CheckIsPrimitiveType(Type type)
{
     return (type == typeof(object) || Type.GetTypeCode(type) != TypeCode.Object);
}
```

**Comment by NickCraver** [comment]
We'd like to see this added as well. We're currently doing custom implementations in Dapper we'd rather avoid.

**Comment by branciat** [comment]
i saw that It's added in the rc2 on myget aspnetcidev repository, i use for target coreclr.

**Comment by pallavit** [comment]
Addressed via #5381
